lr_face/data.py

```diff
@@ -154,6 +154,24 @@ def same_identity(self) -> bool:
         """
         return self.first.identity == self.second.identity
 
+    # Experts' log LLR estimation (optional).
```

Comment:
move to docstring

Reply:
and it is done with the other figures, so not really optional unless you don't want any figures at all